### ISSUE TITLE
Very low FPS when streaming (Windows 11)

### ISSUE THREAD

**Issue by deko0** [Vencord/Vesktop#525]
**Describe the bug**

Trying to stream anything results in having a framerate of about one, maybe less. Switching to a browser, regular discord, or discord with Vencord, the framerate and streams work as expected. Framerate does not fluctuate, it is constantly that.

**To Reproduce**

  1) Try streaming anything, in any resolution/fps combo
_2) My stream is washed 💔_

**Expected behavior**

Stream framerate to be what i choose it to be

**Desktop (please complete the following information):**

Windows 11 23H2

**Command line output**

doesn't do anything on windows, i assume this is for Linux?

**Additional context**

My gpu drivers are updated. Tried reinstalling using DDU. It's not either my connection or my pc. 
Just in case this is needed, here are my specs:
GTX 1060 low profile 3gb
Intel core i5 8400
16GB DDR4-2666mhz ram
Vesktop and Discord installed on an m.2 nvme SSD

**Comment by PolisanTheEasyNick** [comment]
Most probably you are installed Vesktop from Vesktop-Setup-1.5.1.exe installer from Releases, which is not actual upstream version of Vesktop. There was a bug when framerate is 30 fps; resolution picking from display resolution by standard; and bitrate capped to 2500 kbit/s. All this was fixed in #489.
Can you re-produce issue using [building from source](https://github.com/Vencord/Vesktop?tab=readme-ov-file#building-from-source)?

**Comment by deko0** [comment]
tutorial is for Linux. no idea how to those things in windows though

> Most probably you are installed Vesktop from Vesktop-Setup-1.5.1.exe installer from Releases, which is not actual upstream version of Vesktop. There was a bug when framerate is 30 fps; resolution picking from display resolution by standard; and bitrate capped to 2500 kbit/s. All this was fixed in #489. Can you re-produce issue using [building from source](https://github.com/Vencord/Vesktop?tab=readme-ov-file#building-from-source)?

How do i do that stuff on Windows? I cloned the repo but I don't really know how to build it.

**Comment by PolisanTheEasyNick** [comment]
> How do i do that stuff on Windows? I cloned the repo but I don't really know how to build it. and yeah, i did in fact install 1.5.1

You need to install pnpm firstly: https://pnpm.io/installation

**Comment by deko0** [comment]
> > How do i do that stuff on Windows? I cloned the repo but I don't really know how to build it. and yeah, i did in fact install 1.5.1
> 
> You need to install pnpm firstly: https://pnpm.io/installation
yeah i tried that. It complains because the program asks for an older version, 8.11.0

**Comment by deko0** [comment]
> > > You can install version 8.x here: https://pnpm.io/8.x/installation Running the installation command should replace the 9.x you've installed Then you'll just need to follow the build/package guide in the readme
> > 
> > 
> > Okay, I managed to uninstall pnpm and get the right version, and then follow the guide. I managed to install the newer version, I can tell because the stream menu is different.. but its stil laggy, i get like maybe 1 more fps.
> 
> I believe best way for you will be just to install Vencord on official discord

yeah i have vencord but tbh vesktop seems to just run better... how sad

**Comment by Vendicated** [comment]
could you see if this is fixed in v1.5.2?

**Comment by Chemrat** [comment]
I'm on Vesktop 1.5.3 and Windows 11, and experiencing same issue, at least symptomatically
Streaming through official client has no issues

**Comment by streifen-dev** [comment]
Same issue here on Vesktop 1.5.5 with and without hardware acceleration on any combination of stream settings. Curiously the stream goes up to 30fps if I alt-tab out of the game I am trying to stream

**Comment by MemesPwr** [comment]
> I'm on Vesktop 1.5.3 and Windows 11, and experiencing same issue, at least symptomatically Streaming through official client has no issues

same for me on 1.5.5 vesktop  

i'm trying to stream 1080@60 or even 720@30, the result is same, fps is staying at 15-20 but when i'm going to run/walk/whatever, it's gonna drop to 4fps.  

windows 11, build 22631.3737
hw acceleration is enabled in vesktop
game is not matter. for example, `Kingdom Come: Deliverance II`
cpu: r7 2700
gpu: rtx 3070  

and on official client everything is ok for me too  

// that's so sad that there is no way to automatically always bring back vencord to official client :(

**Comment by Mingooberus** [comment]
Having same issue, have built from source and used the release version to no avail.

**Comment by Morphexe** [comment]
Same thing happening on latest for me.

**Comment by Alexanderfem** [comment]
Same for me

**Comment by ItsPasi** [comment]
This issue is still happening. Other people report a framerate of 5 fps for my screenshares. Another issue is that fullscreen applications will not be captured at all.

**Comment by bacca87** [comment]
Vesktop 1.6.1 same issue, very laggy on Windows 11

**Comment by rawpie2** [comment]
i have this same issue
